### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
- Changes to heading capitalization, formatting, punctuation, and contractions.
- Removed one role from the mobile device policy roles section.

### CODE REVIEW

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 1]
```diff
@@ -6,31 +6,33 @@ description: >-
...
+At Mattermost, we want you to be successful from Day 1. As staff members, you're empowered to determine and purchase what you need in order to do your job. We value high impact over stringent expense rules and these expense guidelines are intended to help make decisions not hinder movement. Depending on country/region, some items may be way below or way above the averages listed. If you need to spend more to get the right equipment for you to be productive, we trust you’re making purchase decisions to optimize cost and efficiency. 
 
-## Mattermost Guidelines
+If you're uncertain, feel free to reach out in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel.
```
Comment (jasonblais) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or direct message @isabel.alorro on Mattermost.
```

Comment (jasonblais) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: ^although asking questions in public is encouraged, some may feel more comfortable reaching out directly, especially if they're new staff members

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 2]
```diff
@@ -52,35 +54,34 @@ While we're sharing average costs for various items below, you should use your b
...
-The Mattermost mobile phone policy enables the use of mobile devices for customer-facing roles or those providing repeating on-call service in a secure fashion.
+The Mattermost mobile device policy enables the use of mobile devices for customer-facing roles or those providing repeating on-call service in a secure fashion.
 
 #### Subscription
 
-If you are using your phone primarily for business related matters as defined by your local tax laws, we will reimburse up to 100 USD. Positions or roles where mobile reimbursement may be needed include the following:
+If you are using your device primarily for business-related matters as defined by your local tax laws, we will reimburse up to 100 USD. Positions or roles where mobile reimbursement may be needed include the following:
 
 * Customer-facing roles: 
-    * Account Executives
```
Comment (jasonblais) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md:82]: I believe we do still have AEs as customer-facing role

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 3]
```diff
@@ -137,23 +139,23 @@ Many staff members will use their company issued laptop until it breaks. That sa
...
 
-### MEALS AND ENTERTAINMENT
+### Meals and entertainment
 
-Spend company money like it’s your own. When traveling on behalf of Mattermost, average meal costs will vary by destination city and country. The two links below are helpful reference points for city and country averages. Note: 20% max is considered a gratuity for exceptional service. If helpful, the below links provide information regarding daily per diem guidance in the US as well as other countries.
+Spend company money like it’s your own. When traveling on behalf of Mattermost, average meal costs will vary by destination city and country. The two links below are helpful reference points for city and country averages. Note: 20% is considered the maximum gratuity for exceptional service. 
 
-As an example, daily per diems in the following locations are below:
+The below links provide information regarding daily per diem guidance in the US as well as other countries.
```
Comment (jasonblais) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
The below links provide a few examples regarding daily per diem guidance.
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Looks great! Few suggestions below

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 1 above)
Comment (justinegeffen) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: +1, definitely agree.

**Review by jasonblais — DISMISSED** [review]

**Comment by TQuock** [comment]
I for some reason cannot access github to make amendments. That said, I
have a few revisions to the updates that I'd like to make.


*Tim Quock *
Director of Accounting
tim@mattermost.com | LinkedIn
<https://www.linkedin.com/in/timothy-quock-23300628/> | mattermost.com | google
calendar availability <https://calendly.com/matters/30min-1>

O: (650) 667-8512
C: (925) 878-1338



*"Train people well enough so they can leave, treat people well enough so
they don't want to" - Richard Branson *

**Review by natalie-hub — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Following up on this, @TQuock, when you have a chance. :)

**Comment by justinegeffen** [comment]
@TQuock, following up on this to confirm changes and ready to merge. :)

**Comment by TQuock** [comment]
@justinegeffen can you please see my updated language proposed. I don't want staff just reaching out to Isabel directly. The compromise is adding expenses@mattermost.com. Can you help with getting the changes in the handbook?

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 4]
```diff
@@ -6,31 +6,33 @@ description: >-
...
+At Mattermost, we want you to be successful from Day 1. As staff members, you're empowered to determine and purchase what you need in order to do your job. We value high impact over stringent expense rules and these expense guidelines are intended to help make decisions not hinder movement. Depending on country/region, some items may be way below or way above the averages listed. If you need to spend more to get the right equipment for you to be productive, we trust you’re making purchase decisions to optimize cost and efficiency. 
 
-## Mattermost Guidelines
+If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or direct message @isabel.alorro on Mattermost.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or reach out to expenses@mattermost.com.
```

**Comment by justinegeffen** [comment]
Thanks Tim, I've updated that sentence and committed the change. :)

**Comment by TQuock** [comment]
Thanks Justine!

*Tim Quock *
Director of Accounting
tim@mattermost.com | LinkedIn
<https://www.linkedin.com/in/timothy-quock-23300628/> | mattermost.com | google
calendar availability <https://calendly.com/matters/30min-1>

O: (650) 667-8512
C: (925) 878-1338



*"Train people well enough so they can leave, treat people well enough so
they don't want to" - Richard Branson *

**Comment by justinegeffen** [comment]
@jasonblais, I re-requested your review as I need one approval to merge. :)

**Review by jasonblais — APPROVED** [review]

**Pull request merged**